### PR TITLE
WFP 13103-01 cal csvs (for Pioneer 21)

### PR DESCRIPTION
**Pull request by rtravis-whoi**
For DOFSTK-03412 - originally Sea-Bird was unable to provide SOC Adjust pdf which we use for oxygen_signal_slope cal coeff so I was referencing the cal coeff from the instrument captured during QCT testing.  However (of course), they finally provided it to me the day after I submitted this PR.  So the cal csv has been updated w/ exception notes removed and is a 'normal' DOFSTK cal csv now.

### CODE REVIEW

**Comment by rtravis-whoi** [comment]
[Pioneer 21 AM Updates](https://trello.com/c/65WTFp4J/252-pioneer-21-am-updates)

File: calibration/FLORTK/CGINS-FLORTK-01117__20240226.csv
Comment (reedan88) [inline calibration/FLORTK/CGINS-FLORTK-01117__20240226.csv:1]: Looks good.

File: calibration/PARADK/CGINS-PARADK-20658__20240816.csv
Comment (reedan88) [inline calibration/PARADK/CGINS-PARADK-20658__20240816.csv:1]: Looks good.

**Review by reedan88 — APPROVED** [review]
Looks good.

**Pull request merged**